assets/app/models/Github.js

```diff
@@ -173,16 +186,15 @@ var GithubModel = Backbone.Model.extend({
       return isOfType;
     });
   },
-  addPage: function (opts) {
-    opts = opts || {};
-    var content = (this.configFiles['_defaults.yml'].present) ?
-                    this.configFiles['_defaults.yml'].json : '\n',
-      commitOpts = {
-        path: opts.path,
-        message: opts.message || 'The file ' + opts.path + ' was created',
-        content: opts.content || content
-      };
-    this.commit(commitOpts);
+  getDefaults: function () {
+    var config = this.configFiles['_config.yml'],
+        defaultConfigs = (config.present && config.json) ? config.json.defaults : [],
+        defaults = defaultConfigs.filter(function (c) {
```

Comment:
I think that if condition isn't complete. It should be something like `config.present && config.json && config.json.defaults` but that's pretty verbose. Essentially defaultConfigs should always at least be an empty array.